### PR TITLE
chore: improve auto labeling 🍒

### PR DESCRIPTION
**Pull request by mcdurdin**
Cherry-picked from #7288.

@keymanapp-test-bot skip

### CODE REVIEW

File: .github/multi-labeler.yml [hunk 1]
```diff
@@ -20,26 +22,38 @@ labels:
...
+
+  # Scopes that we look for in the PR title
+
   - label: 'android/'
     matcher:
       title: '\(.*android.*\):'
   - label: 'common/'
     matcher:
       title: '\(.*common.*\):'
+  - label: 'core/'
```
Comment (jahorton) [inline .github/multi-labeler.yml:54]: `core/` does not exist on `stable-15.0`, does it?

Comment (mcdurdin) [reply .github/multi-labeler.yml:54]: The label exists, and the `(core)` title can be used. Just the folder is common/core/desktop/. So I think this labeler rule should be fine?

**Review by darcywong00 — APPROVED** [review]
lgtm

File: .github/multi-labeler.yml (hunk 1 above)
Comment (jahorton) [reply .github/multi-labeler.yml:66]: Ah, right, it's against the title, not the path.  That's what got me.  👍

**Review by jahorton — APPROVED** [review]

**Review by ermshiperete — APPROVED** [review]

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.271](https://keyman.com/downloads/releases/stable/15.0.271)

**Pull request merged**